app/helpers/dashboards_helper.rb

```diff
@@ -16,9 +16,10 @@ def voicemail_options
     enum_text = { not_specified: 'No instructions; no ID VM',
                   no: 'Do not leave a voicemail',
                   yes: 'Voicemail OK, ID OK' }
-    vm_options = Patient::VOICEMAIL_PREFERENCE
+    vm_options = Patient.voicemail_preference.values
```

Comment:
Want to make sure I follow this -- we take voicemail pref options, change them to symbols (because they're being returned a strings), and then turn it into a tuple of the text and symbol. 

Wonder if we can ditch that second `to_sym`, or collapse lines 19 and 20?